src/Solver/time_stepping/TimeCluster.cpp

```diff
@@ -495,7 +495,7 @@ void seissol::time_stepping::TimeCluster::computeNeighboringIntegration( seissol
   CellLocalInformation*       cellInformation               = i_layerData.var(m_lts->cellInformation);
 #ifdef USE_PLASTICITY
   PlasticityData*             plasticity                    = i_layerData.var(m_lts->plasticity);
-  real                      (*energy)[3]                    = i_layerData.var(m_lts->energy);
+  //real                      (*energy)[3]                    = i_layerData.var(m_lts->energy);
```

Comment:
Please remove the whole line instead of commenting.

Could you also try to remove
Variable<real[3]> energy;
and
tree.addVar( energy, plasticityMask, PAGESIZE_HEAP, seissol::memory::Standard );
from Initializers/LTS.h?

It seems that they are never used within SeisSol.